lms/djangoapps/discussion/rest_api/tests/utils.py

```diff
@@ -675,7 +675,7 @@ def register_get_username_replacement_response(self, user, status=200, body=""):
         self.set_mock_return_value('update_username', body)
 
     def register_subscribed_threads_response(self, user, threads, page, num_pages):
-        self.set_mock_return_value('get_user_subscriptions', {
+        self.set_mock_return_value('get_user_threads', {
```

Comment:
Why was this change necessary?

Reply:
We missed moving get_user_subscriptions, I've created another PR for it.
https://github.com/edly-io/edx-platform/commit/12fc34105c19d8ddc10b96e3d37b96da311d6285